rest-client-quickstart/src/main/docker/Dockerfile.jvm

```diff
@@ -81,6 +81,7 @@ FROM registry.access.redhat.com/ubi8/openjdk-21:1.20
 
 ENV LANGUAGE='en_US:en'
 
+ENV HOST 0.0.0.0
```

Comment:
Is this not the default?

Reply:
I think so, it was added just to make sure, i will remove it and hope nothing breaks

Reply:
If it deploys it should be fine